### PR TITLE
CHROMEOS build-configs.yaml: Enable kernel serial console

### PR DESCRIPTION
**Pull request by nuclearcat**
By default, with chromeos fragments CONFIG_SERIAL_8250_CONSOLE is disabled,
while it is important to have this option enabled to monitor kernel messages
in LAVA.

Signed-off-by: Denys Fedoryshchenko <denys.f@collabora.com>

### CODE REVIEW

**Comment by nuclearcat** [comment]
This is old job: https://lava.collabora.co.uk/scheduler/job/6482968
This is with serial console enabled: https://lava.collabora.co.uk/scheduler/job/6485369
As collateral, because it generate more data for console it helps to detect "login:" keyword

**Comment by gctucker** [comment]
LGTM, and we should also be able to merge this in `main` once it's merged in `chromeos`.

**Review by mgalka — APPROVED** [review]
LGTM

**Pull request merged**
